src/ralph/data_importer/tests.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+from six import text_type
+
+from django.core import management
+from django.test import TestCase
+
+from ralph.assets.models.assets import (
+    AssetModel,
+    Environment,
+    Service,
+    ServiceEnvironment
+)
+from ralph.assets.models.choices import ObjectModelType
+from ralph.back_office.models import (
+    BackOfficeAsset,
+    Warehouse
+)
+from ralph.data_importer.management.commands import importer
+from ralph.data_importer.resources import AssetModelResource
+
+
+class DataImporterTestCase(TestCase):
+
+    """TestCase data importer command."""
+
+    warehouse_csv = 'warehouse.csv'
+    back_office_csv = 'back_office.csv'
+
+    @classmethod
+    def setUpClass(cls):
+        asset_model = AssetModel()
+        asset_model.name = "asset_model_1"
+        asset_model.type = ObjectModelType.back_office
+        asset_model.save()
+
+        warehouse = Warehouse()
+        warehouse.name = "warehouse_1"
+        warehouse.save()
+
+        environment = Environment()
+        environment.name = "environment_1"
+        environment.save()
+
+        service = Service()
+        service.name = "service_1"
+        service.save()
+
+        service_environment = ServiceEnvironment()
+        service_environment.environment = environment
+        service_environment.service = service
+        service_environment.save()
+
+        with open(cls.warehouse_csv, 'w+') as f:
```

Comment:
(question) do we want to fill these files from python code or could we prepare these files in samples dir and only load them here? @ar4s @xliiv